.pyup.yml

```diff
@@ -1 +1,6 @@
 schedule: "every week on monday"
+requirements:
+  - requirements/flake8.txt
+  - requirements/py27-tests.txt
+  - requirements/py36-tests.txt
```

Comment:
These are already discovered by pyup.io's [file search](https://pyup.io/docs/bot/config/#file-search) so there's no need to list them explicitly. To avoid needing to maintain this list, let's revert this change. In fact, we could simply move `tox.txt` to `requirements/tox.txt` and we wouldn't need to change this file at all.